abp168/shadows_of_knight_ep1_(Dr.Jones_Comments).py

```diff
@@ -0,0 +1,119 @@
+# *************************************************
```

Comment:
Combine this file the file above, then delete this file.

Reply:
Combined and deleted